### PR TITLE
Customer service branch

### PR DESCRIPTION
**Pull request by KajalMathur**
Kindly review my code , made changes as per your review comments in the last PR.

### CODE REVIEW

File: src/test/java/com/customer/customerService/CustomerServiceApplicationTests.java [hunk 1]
```diff
@@ -1,16 +1,90 @@
...
+		customer.add(c2);
+		when(customerRepository.findAll()).thenReturn(customer);
+		when(customerResponse.setStatus(date, experienceDate)).thenReturn(CustomerResponse.Status.NEW);
+		when(customerResponse.getStatus()).thenReturn(CustomerResponse.Status.NEW);
+
+		// When
+		List<CustomerResponse> customerResponseList = customerServiceImpl.getAllCustomers();
+
+		// Then
+		System.out.print("Joining and expiry date=" + customerResponseList.get(1).getJoiningDate() + " "
```
Comment (AakashIsTheLimit) [inline src/test/java/com/customer/customerService/CustomerServiceApplicationTests.java]: no system.out statements in PR please.

Please make sure you always get rid of these debugging statements before creating a new revision for the PR

File: src/main/java/com/customer/service/CustomerServiceImpl.java [hunk 2]
```diff
@@ -9,69 +10,59 @@
...
+		if (customer.isPresent()) {
+			CustomerResponse customerResponse = CustomerResponse.builder().id(customer.get().getId())
+					.firstName(customer.get().getFirstName()).lastName(customer.get().getLastName())
+					.joiningDate(customer.get().getJoiningDate()).expiryDate(customer.get().getExpiryDate())
+					.address(customer.get().getAddress()).build();
+			customerResponse.setStatus(customer.get().getJoiningDate(), customer.get().getExpiryDate());
+			return customerResponse;
+		}
+		else
+			throw new CustomerNotFoundException("");
```
Comment (AakashIsTheLimit) [inline src/main/java/com/customer/service/CustomerServiceImpl.java]: There should be some error message here.

Comment (AakashIsTheLimit) [reply src/main/java/com/customer/service/CustomerServiceImpl.java]: Where is @ControllerAdvise being used ? I dont see the definition anywhere

File: src/test/java/com/customer/customerService/CustomerServiceApplicationTests.java (hunk 1 above)
Comment (KajalMathur) [reply src/test/java/com/customer/customerService/CustomerServiceApplicationTests.java]: done

File: src/main/java/com/customer/service/CustomerServiceImpl.java (hunk 2 above)
Comment (KajalMathur) [reply src/main/java/com/customer/service/CustomerServiceImpl.java]: done , was not commited.

**Pull request closed without merging**